specification/components/schemas/signedContractData.v1.json

```diff
@@ -23,9 +23,10 @@
         "cpsV2GRootCertificateFingerprint": {
             "$ref": "../parameters/cpsV2GRootCertificateFingerprintParam.v1.json"
         },
-        "cpsRootDistinguishedName": {
```

Comment:
the param name should stay "cpsRootDistinguishedName" and not "cpsV2GRootCertificateFingerprint" which is described above (copy paste error i think)